docker/irods_client/tests/test_session.py

```diff
@@ -2,6 +2,20 @@
 
 from ibridges import Session
 
+def test_session_from_cached_pw(config, irods_env):
+    # test only for plain irods
+    if not "set_home_perm" in config:
```

Comment:
Actually this test should be skipped for the Yoda environment. So I will use the "environment" setting in the config. But then it would also be good to keep its couterpart in the plain iROPDS settings.

Reply:
I think it's better long term if we don't test for the environment. Simply, because we will/might need to change the tests every time we add a new environment to our test suite. My suggestion would be something like using "xfail_iinit_session". If we put all the xfails in the toml file, it is also easy to see which features do not work/are broken.